docs/data/pages.ts

```diff
@@ -364,7 +369,7 @@ const pages: MuiPage[] = [
   },
   {
     pathname: '/x/migration-group',
-    title: 'Migration',
+    title: 'Migration Guides',
```

Comment:
MUI Core is also using `Migration` as the title for this group/section. 🤔 
Maybe we should stay in sync and update both or none? 🤔

Reply:
Fair point. Let's not update this now.
Maybe @samuelsycamore can consider if it's worth doing the change throughout the products.